frontend/src/components/pages/CampCreation/SessionDayButton.tsx

```diff
@@ -0,0 +1,28 @@
+import { IconButton, Text } from "@chakra-ui/react";
+import React from "react";
+
+type SessionDayButtonProps = {
+  day: string;
+  active: boolean | undefined;
+  onSelect?: (day: string) => void;
```

Comment:
Sorry, missing something -- why is this optional?

Reply:
didn't think I'd need this to have on click for the cards but will probably need it - will remove the optional